index.js

```diff
@@ -853,9 +933,9 @@ sockserve.on('connection', function(conn) {
 		// it should be an auth message
 		conn.removeListener("message", func);
 		var s = message.split(":");
-		if((s.length === 4 || s.length === 2) && s[0] === "auth") {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2591098533/issues/source?bid=3248399&fileBranchId=3306078#l936)